### ISSUE TITLE
Warnings generated when compiling with Clang

### ISSUE THREAD

**Issue by kartikkumar** [SRombauts/SQLiteCpp#69]
Hi all,

Just thought I'd share these warnings that pop up when compiling with the Clang compiler (Apple LLVM version 7.0.0 (clang-700.0.72) on Mac OS X 10.11. They're related to Googletest and so I'm guessing they're benign. Just thought it would be good to share this for the record.

```
In file included from SQLiteCpp/googletest/include/gtest/gtest.h:58:
SQLiteCpp/googletest/include/gtest/internal/gtest-internal.h:1029:68: warning: commas at the end of enumerator
      lists are a C++11 extension [-Wc++11-extensions]
        Element, GTEST_REMOVE_REFERENCE_AND_CONST_(Element)>::value,
                                                                   ^
In file included from SQLiteCpp/googletest/src/gtest-all.cc:42:
SQLiteCpp/googletest/src/gtest.cc:374:12: warning: missing field 'owner_' initializer
      [-Wmissing-field-initializers]
GTEST_API_ GTEST_DEFINE_STATIC_MUTEX_(g_linked_ptr_mutex);
           ^
SQLiteCpp/googletest/include/gtest/internal/gtest-port.h:1905:80: note: expanded from macro
      'GTEST_DEFINE_STATIC_MUTEX_'
     ::testing::internal::MutexBase mutex = { PTHREAD_MUTEX_INITIALIZER, false }
                                                                               ^
2 warnings generated.

In file included from SQLiteCpp/googletest/include/gtest/gtest.h:58:
SQLiteCpp/googletest/include/gtest/internal/gtest-internal.h:1029:68: warning: commas at the end of enumerator
      lists are a C++11 extension [-Wc++11-extensions]
        Element, GTEST_REMOVE_REFERENCE_AND_CONST_(Element)>::value,
                                                                   ^
1 warning generated.

In file included from SQLiteCpp/googletest/include/gtest/gtest.h:58:
SQLiteCpp/googletest/include/gtest/internal/gtest-internal.h:1029:68: warning: commas at the end of enumerator
      lists are a C++11 extension [-Wc++11-extensions]
        Element, GTEST_REMOVE_REFERENCE_AND_CONST_(Element)>::value,
                                                                   ^
1 warning generated.

In file included from SQLiteCpp/googletest/include/gtest/gtest.h:58:
SQLiteCpp/googletest/include/gtest/internal/gtest-internal.h:1029:68: warning: commas at the end of enumerator
      lists are a C++11 extension [-Wc++11-extensions]
        Element, GTEST_REMOVE_REFERENCE_AND_CONST_(Element)>::value,
                                                                   ^
1 warning generated.
```

Kind regards,

Kartik

**Comment by SRombauts** [comment]
Thanks for reporting!

**Comment by SRombauts** [comment]
Fix with commit a573c8c where I deactivated some not so useful specific warnings
